libraries/src/Form/Field/ImagelistField.php

```diff
@@ -3,43 +3,61 @@
 /**
  * Joomla! Content Management System
  *
- * @copyright  (C) 2010 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ * @copyright (C) 2010 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
```

Comment:
```suggestion
 * @copyright  (C) 2010 Open Source Matters, Inc. <https://www.joomla.org>
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```
Please revert your change or accept my suggestion which does the same. We require 2 spaces between the longest tag and the text.